src/assets/_project/_blocks/utils/qg-misc.js

```diff
@@ -25,21 +25,20 @@
   });
   // this function equals the height of the cards in a group, if it finds a class '.cards__equal-height'.
   function setHeight() {
-    if ($('.cards__equal-height').length > 0) {
```

Comment:
Are we likely to be running this in a situation where jQuery isn't available?

Reply:
Not in 4.3. Unsure about future versions.  Preference is to remove dependency on external libraries if they are not required.  Vanilla JS can handle this easily enough.